.travis.yml

```diff
@@ -27,6 +27,10 @@ env:
   - PKG=elementsd STABLE=0
   - PKG=electrs STABLE=1
   - PKG=electrs STABLE=0
+  - pkg=nbxplorer STABLE=1
+  - pkg=nbxplorer STABLE=0
+  - pkg=btcpayserver STABLE=1
+  - pkg=btcpayserver STABLE=0
```

Comment:
we won't need this if btcpayserver gets merged upstream

Reply:
Noted